knowledge-base/scheduler-kb-context-menu-on-appointments-and-empty-slots.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Manage Appointments via Context Menu in the Scheduler
+description: How to use the context menu to add, edit, delete appointments in specific timeslots in the Scheduler.
+type: how-to
+page_title: How to Add Appointments on Empty Slot in the Scheduler Using the ContextMenu. How to Add ContextMenu on Appointment in the Scheduler.
+slug: scheduler-kb-context-menu-on-appointments-and-empty-slots
+tags: scheduler, blazor, contextmenu, appointments, empty slots
```

Comment:
Consider adding a previous_url field to handle the deleted file and any possible 404s from it.